README.md

```diff
@@ -110,7 +110,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link]( https://brightbrain20.github.io/Portfolio/)
 
```

Comment:
- [ ] Well done @brightbrain20, your online version works fine, and you have completed the milestone, however, if I may please, I have an optional suggestion to raise regarding your desktop animation, I saw that is a background color change effect on a scroll at the hero section, that is an awesome idea but I do not think those colors match the design of your app, the yellow in red and a blue again humm, in my opinion, it is looking... you know the work 😄, you can keep the effect but make sure to choose different colors and reduce the opacity because they are much bright.

![5](https://user-images.githubusercontent.com/77843050/208385626-fcb0106c-370c-4eb0-be8b-cf821bdf6ba9.gif)